gbfs.md

```diff
@@ -84,23 +100,60 @@ geofencing_zones.json <br/>*(added in v2.1-RC)* | Optional | Geofencing zones an
+* Files are distributed as individual HTTP endpoints.
+    * Required files must not 404. They must return a properly formatted JSON file as defined in [Output Format](#output-format).
     * Optional files may 404. A 404 of an optional file should not be considered an error.
-* Auto-Discovery:
-    * This specification supports auto-discovery.
-    * The location of the auto-discovery file will be provided in the HTML area of the shared mobility landing page hosted at the URL specified in the URL field of the system_infomation.json file.
-    * This is referenced via a _link_ tag with the following format:
+    
+### Auto-Discovery 
+Publishers should implement auto-discovery of GBFS feeds by linking to the location of the `gbfs.json` auto-discovery endpoint.
+ * The location of the auto-discovery file should be provided in the HTML area of the shared mobility landing page hosted at the URL specified in the URL field of the `system_infomation.json` file.
+
+ * This is referenced via a _link_ tag with the following format:
       * `<link rel="gbfs" type="application/json" href="https://www.example.com/data/gbfs.json" />`
     * References:
       * http://microformats.org/wiki/existing-rel-values
       * http://microformats.org/wiki/rel-faq#How_is_rel_used
-
+ * A shared mobility landing page may contain links to auto-discovery files for multiple systems.
 ### Localization
 * Each set of data files should be distributed in a single language as defined in system_information.json.
 * A system that wants to publish feeds in multiple languages should do so by publishing multiple distributions, such as:
     * `https://www.example.com/data/en/system_information.json`
     * `https://www.example.com/data/fr/system_information.json`
+    
+### Text Fields and Naming
+
+Rich text should not be stored in free form text fields. Fields should not contain HTML.
+
+All customer-facing text strings (including station names) should use Mixed Case (not ALL CAPS), following local conventions for capitalization of place names on displays capable of displaying lower case characters.
+    
+   * Examples:
+     * Central Park South
+     * Villiers-sur-Marne
+     * Market Street
+
+Abbreviations should not be used for names and other text (e.g. St. for Street) unless a location is called by its abbreviated name (e.g. “JFK Airport”). Abbreviations may be problematic for accessibility by screen reader software and voice user interfaces. Consuming software can be engineered to reliably convert full words to abbreviations for display, but converting from abbreviations to full words is prone to more risk of error.
+
+Names used for stations, virtual stations and geofenced areas should be human readable. Naming conventions used for locations should consider a variety of use cases including both text and maps. 
+
+Descriptions should not include information so specific that it could be used in tracking of vehicles or trips.
```

Comment:
I believe this originally came up in reference to car sharing. I know there are bike systems where each individual bike has a unique name which becomes the equivalent of a bike ID that would allow for the recreation of trips. In car sharing it would be using
`"vehicle_desc": "Four-door black sedan" `  
in stead of
 `"vehicle_desc": "Four-door black sedan #3" `